docs/spec/spec.md

```diff
@@ -0,0 +1,309 @@
+| Create | The Provisioner referenced will take ownership of the Channel and begin provisioning the backing resources required for the Channel depending on implementation. | Only one Provisioner is allowed to be the Owner for a given Channel. |
+| Update | The Provisioner will synchronize the Channel backing resources to reflect the update. | |
+| Delete | The Provisioner will deprovision the backing resources if no longer required depending on implementation. | |
+
+---
+
+## kind: Provisioner
+
+### group: eventing.knative.dev/v1alpha1
+
+_Describes an abstract configuration of a Source system which produces events
+or a Channel system that receives and delivers events._
+
+### Object Schema
+
+#### Spec
+
+| Field | Type | Description | Limitations |
+| --- | --- | --- | --- |
+| type*| TypeMeta<sup>1</sup> | The type of the resource to be provisioned. | Must be Source or Channel. |
+
+\*: Required
+
+1: Kubernetes type.
+
+#### Metadata
+
+##### Owner References
+
+ * Owns EventTypes.
+
+#### Status
+
+| Field | Type | Description | Limitations |
+| --- | --- | --- | --- |
+| provisioned |[]ProvisionedObjectStatus| Status of creation or adoption of each EventType and errors therein. | It is expected that a provisioner list all produced EventTypes, if applicable. |
+
+#### Events
+
+ * Source created
+ * Source deleted
+ * Event types installed
+
+### Life Cycle
+
+| Action | Reactions | Limitations |
+| --- | --- | --- |
+| Create | Creates and owns EventTypes produced, or adds Owner ref to existing EventTypes. | Verifies Json Schema provided by existing EventTypes; Not allowed to edit EventType if previously Owned; |
+| Update | Synchronizes EventTypes. | |
+| Delete | Removes Owner ref from EventTypes. | |
+
+---
+
+## kind: Subscription
+
+### group: eventing.knative.dev/v1alpha1
+
+_Describes a direct linkage between an event publisher and an action._
+
+### Object Schema
+
+#### Spec
+
+| Field | Type | Description | Limitations |
+| --- | --- | --- | --- |
+| from*| ObjectRef | The originating Channel for the link. | Must be a Channel. |
+| call<sup>1</sup> | EndpointSpec | Optional processing on the event. The result of call will be sent to result. | |
+| result<sup>1</sup> | ObjectRef | The continuation Channel for the link. | Must be a Channel. |
+
+\*: Required
+
+1: At Least One(call, result)
+
+#### Metadata
+
+##### Owner References
+
+ * If a resource controller created this Subscription: Owned by the originating resource.
+
+##### Conditions
+
+ * **Ready.**
+ * **FromReady.**
+ * **CallActive.** True if the call is sinking events without error.
+ * **Resolved.**
+
+#### Events
+
+ * PublisherAcknowledged
+ * ActionFailed
+
+### Life Cycle
+
+| Action | Reactions | Limitations |
+| --- | --- | --- |
+| Create | The publisher referenced needs to be watching for Subscriptions. | |
+| Update | | |
+| Delete | | |
+
+---
+
+
+## kind: EventType
+
+NOTE: EventType is out of scope for 0.1 release. This is future documentation.
```

Comment:
@evankanderson fight!

Reply:
I'd agree that if something is not in scope, it should not be on the spec for that particular release.

Perhaps worth to shuffle around and introduce some "future goals" doc? Things out of scope could be parked there?

Reply:
I think the idea was to mark out some additional "planned but not yet implemented" features that we know we want to do, but don't have time for the 0.1 release.

Ideally, the spec would give us some runway to code against past the 0.1 release, rather than being a point-in-time snapshot of the `api/*/*.go` files in a different form.

Reply:
Recommend moving this to the end, though.

Reply:
At least so far, the implementation of these resources has been the easy part and the agreeing on the spec the hard part. If EventType remains in the spec, we're going to discuss it. I think there are more important places to focus attention.